### ISSUE TITLE
Help me, free membership wont stay when i log off

### ISSUE THREAD

**Issue by Acht-the-Deadf1sh** [rxzyx/prodigy-hack#120]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by Scriptonotics** [comment]
Yeah, it is mean't to be like that becuase when were you leave restart or shut down the window the Inspect console dose not save the data and reset as it was before you copy pasted the Membership hack in the console. 

So, you may need to come back here and copy paste the Membership hack again each and every time you log on to Prodigy and want Memebership.

I hope that was very helpful to you and the other with this issue.
